src/components/Cards/CardHeightMatching.test.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { ResizeSensor, mockDetach } from 'css-element-queries';
+import CardHeightMatching from './CardHeightMatching';
+import * as helpers from '../../common/helpers';
+
+jest.mock('css-element-queries');
+
+jest.spyOn(helpers, 'debounce').mockImplementation(v => v);
+const mockQuerySelectorAll = jest.fn();
+HTMLElement.prototype.querySelectorAll = mockQuerySelectorAll;
+
+const mockElement = {
+  scrollHeight: 0,
+  style: {
+    height: 0
+  }
+};
+
+const props = {
+  selector: ['first', 'second'],
+  children: 'children'
+};
+
+beforeEach(() => {
+  mockDetach.mockClear();
+  ResizeSensor.mockClear();
+  helpers.debounce.mockClear();
+  mockQuerySelectorAll.mockReturnValue([mockElement]);
+});
+
+test('creates a ResizeSensor for each selector', () => {
+  mount(<CardHeightMatching {...props} />);
```

Comment:
`mount` was used instead of `shallow` since refs are used inside of the component.